prep-css-layout-classes/index.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Layout Classes</title>
+  <link rel="stylesheet" href="reset.css">
+  <style>
+    /* stylelint-disable block-no-empty */
+     body {
+      font-family: sans-serif;
+    }
+
+    h1 {
+      text-align: center;
+    }
+   #movies {
+     display: flex;
+     padding: 0rem;
+     justify-content: center;
+  }
+  .jones{
+      text-align: justify;
+      display: inline-block;
```

Comment:
You should be able to leave out a `display` property here and still have the code work.